pom.xml

```diff
@@ -1,88 +1,96 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
         <version>4.15</version>
-        <relativePath />
+        <relativePath/>
     </parent>
-    <groupId>io.jenkins.plugins</groupId>
-    <artifactId>minio</artifactId>
-    <version>1.1.1-SNAPSHOT</version>
-    <packaging>hpi</packaging>
+
     <properties>
-        <jenkins.version>2.243</jenkins.version>
+        <revision>1.1.1</revision>
+        <changelist>-SNAPSHOT</changelist>
+        <gitHubRepo>jenkinsci/minio-plugin</gitHubRepo>
+        <jenkins.version>2.249.3</jenkins.version>
+        <bom.artifactId>2.249.x</bom.artifactId>
+        <bom.version>20</bom.version>
         <java.level>8</java.level>
         <powermock.version>2.0.2</powermock.version>
```

Comment:
I was not sure if the explicit Powermock version was still needed so I left it in, but everything compiled and tested fine when I tried removing it :thinking:. Maybe you can tell me if and why this was/is needed?

Reply:
Yeah, I think it's a remnant of when I wanted to write tests for certain parts of the plugin. But since writing tests for Jenkins plugin is a study on its own I forfeited that for now.
I think it's safe to just remove it.

Reply:
I agree 100% on the strange since that is writing tests for Jenkins plugins :smiley: